python/ray/data/read_api.py

```diff
@@ -130,6 +132,14 @@ def from_items(
             on how to tune it, see
             :ref:`Tuning read parallelism <read_parallelism>`.
             Parallelism is upper bounded by ``len(items)``.
+        concurrency: The maximum number of Ray tasks to run concurrently. Set this
+            to control number of tasks to run concurrently. This doesn't change the
+            total number of tasks run or the total number of output blocks. By default,
+            concurrency is dynamically decided based on available resource.
+        override_num_blocks: Override the number of output blocks of read tasks. By
```

Comment:
Not sure how we might re-word this, but from my initial reading it's unclear if this refers to the number of output blocks per read tasks or across all read tasks.

Reply:
Yeah, sgtm